src/System.Text.Json/src/System/Text/Json/Node/JsonNumber.cs

```diff
@@ -0,0 +1,640 @@
+        /// </exception>
+        /// <exception cref="FormatException">
+        ///   <see cref="JsonNumber"/> represents a number in a format not compliant with <see cref="int"/>.
+        /// </exception>
+        public int GetInt32() => int.Parse(_value);
+
+        /// <summary>
+        ///   Converts the numeric value of this instance to its <see cref="long"/> equivalent.
+        /// </summary>
+        /// <returns>A <see cref="long"/> equivalent to the number stored by this instance.</returns>
+        /// <exception cref="OverflowException">
+        ///   <see cref="JsonNumber"/> represents a number less than <see cref="long.MinValue"/> or greater than <see cref="long.MaxValue"/>.
+        /// </exception>
+        /// <exception cref="FormatException">
+        ///   <see cref="JsonNumber"/> represents a number in a format not compliant with <see cref="long"/>.
+        /// </exception>
+        public long GetInt64() => long.Parse(_value);
+
+        /// <summary>
+        ///   Converts the numeric value of this instance to its <see cref="float"/> equivalent.
+        /// </summary>
+        /// <returns>A <see cref="float"/> equivalent to the number stored by this instance.</returns>
+        /// <exception cref="OverflowException">
+        ///   <see cref="JsonNumber"/> represents a number less than <see cref="float.MinValue"/> or greater than <see cref="float.MaxValue"/>.
+        /// </exception>
+        /// <exception cref="FormatException">
+        ///   <see cref="JsonNumber"/> represents a number in a format not compliant with <see cref="float"/>.
+        /// </exception>
+        /// <remarks> 
+        ///   On .NET Core this method does not return <see langword="false"/> for values larger than 
+        ///   <see cref="float.MaxValue"/> (or smaller than <see cref="float.MinValue"/>), 
+        ///   instead <see langword="true"/> is returned and <see cref="float.PositiveInfinity"/> (or 
+        ///   <see cref="float.NegativeInfinity"/>) is emitted. 
+        /// </remarks> 
+        public float GetSingle() => float.Parse(_value);
+
+        /// <summary>
+        ///   Converts the numeric value of this instance to its <see cref="double"/> equivalent.
+        /// </summary>
+        /// <returns>A <see cref="double"/> equivalent to the number stored by this instance.</returns>
+        /// <exception cref="OverflowException">
+        ///   <see cref="JsonNumber"/> represents a number less than <see cref="double.MinValue"/> or greater than <see cref="double.MaxValue"/>.
+        /// </exception>
+        /// <exception cref="FormatException">
+        ///   <see cref="JsonNumber"/> represents a number in a format not compliant with <see cref="double"/>.
+        /// </exception>
+        /// <remarks> 
+        ///   On .NET Core this method does not return <see langword="false"/> for values larger than 
+        ///   <see cref="float.MaxValue"/> (or smaller than <see cref="float.MinValue"/>), 
```

Comment:
double.MaxValue, not float. Same for the Try method
```suggestion
        ///   <see cref="double.MaxValue"/> (or smaller than <see cref="double.MinValue"/>), 
```